pkg/controllers/osc/resources/operating_system_config.go

```diff
@@ -192,6 +194,7 @@ func GenerateOperatingSystemConfig(
 		kubeletConfig:              kubeletConfigs,
 		BootstrapKubeconfig:        bootstrapKubeconfigString,
 		bootstrapConfig:            bc,
+		NetworkIPFamily:            string(networkIPFamily),
```

Comment:
nit: `NetworkIPFamily` -> `IPFamily`.

Reply:
`string(networkIPFamily),` could be just `string(providerConfig.Network.GetIPFamily())`.